### DISCUSSION TITLE
Add possibility to capture stream

### DISCUSSION THREAD

**Discussion by johannbuscail** [iddan/react-native-canvas#273] [Ideas]
Please add the possibility to capture canvas stream:
[Link to MDN doc](https://developer.mozilla.org/en-US/docs/Web/API/MediaStream_Recording_API)
Thanks !

**Comment by KendrickChou** [comment]
PleazPleaz

**Comment by ihorsavka** [comment]
/Is there a chance we will get this functionality?
